documentation/samples/packs/card/weather.ts

```diff
@@ -0,0 +1,100 @@
+import * as coda from "@codahq/packs-sdk";
+export const pack = coda.newPack();
+
+// Define the schema that will be used to render the card.
+const WeatherSchema = coda.makeObjectSchema({
+  properties: {
+    summary: { type: coda.ValueType.String, fromKey: "shortForecast" },
+    forecast: { type: coda.ValueType.String, fromKey: "detailedForecast" },
+    temperature: { type: coda.ValueType.String },
+    wind: { type: coda.ValueType.String, fromKey: "windSpeed" },
+    icon: {
+      type: coda.ValueType.String,
+      codaType: coda.ValueHintType.ImageReference,
+    },
+    link: {
+      type: coda.ValueType.String,
+      codaType: coda.ValueHintType.Url,
+    },
+  },
+  displayProperty: "summary",
+  subtitleProperties: [
+    // Only show the value for the temperature property.
+    { property: "temperature", label: `${coda.PropertyLabelValueTemplate}` },
```

Comment:
oh interesting i guess this effectively becomes the same as label: '' didnt even realize that

Reply:
Oh ya, I was using this when an empty label wasn't working yet. Changed to an empty label now.